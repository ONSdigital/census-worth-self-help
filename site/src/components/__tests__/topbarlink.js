import React from "react"
import renderer from "react-test-renderer"

import TopBarLink from "../topbarlink"

describe("TopBarLink", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<TopBarLink title="menu" link="menu" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})