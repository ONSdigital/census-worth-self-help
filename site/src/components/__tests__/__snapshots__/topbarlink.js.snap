// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBarLink renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "1sm5jo",
      "next": undefined,
      "styles": "
        
    display: flex;
    justify-content: center;
    flex-direction: column;
  
        padding: 0px 10px;
        background-color: rgb(255, 255, 255, 0.2);
        border-radius: 4px;
        margin: 10px 10px 10px 0px;
        min-width: 20px;
        object-fit: contain;
        text-align: center;
        height: 40px;
      ",
    }
  }
>
  <a
    className="Header-button-Style"
    css={
      Object {
        "map": undefined,
        "name": "id1ekf",
        "next": undefined,
        "styles": "
          text-decoration: none;
        ",
      }
    }
    data-test="topbarlink__menu-button"
    href="menu"
  >
    menu
  </a>
</div>
`;
